app/src/main/java/com/example/pinterbest/HomeFragment.kt

```diff
@@ -0,0 +1,74 @@
+package com.example.pinterbest
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.databinding.DataBindingUtil
+import androidx.fragment.app.Fragment
+import androidx.navigation.NavController
+import androidx.navigation.fragment.NavHostFragment
+import androidx.recyclerview.widget.GridLayoutManager
+import com.example.pinterbest.adapters.PinFeedHomeAdapter
+import com.example.pinterbest.data.PinObject
+import com.example.pinterbest.databinding.FragmentHomeBinding
+
+class HomeFragment : Fragment() {
+    private var _binding: FragmentHomeBinding? = null
+    private val binding get() = _binding!!
```

Comment:
take a look on `lateinit var`, it's great approach to avoid force unwrap `!!` and keep clean solution in this case

Reply:
Fixed here: https://github.com/enginire4ek/Pinterbest/pull/1/commits/c349b66d96a4b6bbf03557b4bc3771de1638810b